src/Microsoft.AspNetCore.Server.Testing/Deployers/IISExpressDeployer.cs

```diff
@@ -34,21 +34,23 @@ public override DeploymentResult Deploy()
                 DotnetPublish();
             }
 
+            var webRootLocation = DeploymentParameters.PublishApplicationBeforeDeployment ? DeploymentParameters.PublishedApplicationRootPath : DeploymentParameters.ApplicationPath;
```

Comment:
ah, you mean this should be named something like `appRoot` instead and rest of the changes are good?